integration/testdata/job-master.data

```diff
@@ -100,9 +100,11 @@
 [WARN] 1.7.7 Do not admit containers with dangerous capabilities (Not Scored)
 
 == Remediations ==
-1.1.1 Edit the API server pod specification file /etc/kubernetes/manifests/kube-apiserver.yaml
-on the master node and set the below parameter.
---anonymous-auth=false
+1.1.1 audit test did not run: failed to run: /bin/ps -ef | grep kube-apiserver | grep -v grep, command: [grep kube-apiserver], error: fork/exec /usr/bin/grep: no such file or directory
+failed to run: /bin/ps -ef | grep kube-apiserver | grep -v grep, command: [grep -v grep], error: fork/exec /usr/bin/grep: no such file or directory
+failed to run: /bin/ps -ef | grep kube-apiserver | grep -v grep, command: [/bin/ps -ef], error: write |1: broken pipe
+failed to run: /bin/ps -ef | grep kube-apiserver | grep -v grep, command: [grep kube-apiserver], error: exec: not started
+failed to run: /bin/ps -ef | grep kube-apiserver | grep -v grep, command: [grep -v grep], error: exec: not started
```

Comment:
I don't think we should be expecting to see "audit test did not run" errors like this in the output. It turns out this has revealed a bigger problem!  This test (and others) aren't running correctly because of #574. @robertojrojas is going to get #574 sorted out and then you will need to correct the job*data files for integration tests.